graphql-jpa-query-schema/src/main/java/com/introproventures/graphql/jpa/query/schema/impl/QraphQLJpaBaseDataFetcher.java

```diff
@@ -146,7 +137,7 @@ public Object get(DataFetchingEnvironment environment) {
                 Field selectedField = (Field) selection;
 
                 // "__typename" is part of the graphql introspection spec and has to be ignored by jpa
-                if(!TYPENAME.equals(selectedField.getName())) {
+                if(!TYPENAME.equals(selectedField.getName()) && !CashGraphQLCalcFields.isCalcField(from.getJavaType(), selectedField.getName())) {
```

Comment:
You could use JPA `@Transient` annotation to ignore calculated fields in queries.